src/diagnostics.rs

```diff
@@ -67,14 +76,21 @@ pub fn report_error<'tcx, 'mir>(
                 ExperimentalUb { .. } =>
                     "Undefined Behavior",
                 Deadlock => "deadlock",
+                MultipleSymbolDefinitions { .. } =>
+                    "multiple definitions of",
```

Comment:
of what?

Oh I see what you are doing, you are shoehorning this into the `format!("{}: {}", title, msg)` below. I'm not a fan of that, the individual strings become entirely meaningless.^^

Maybe `title` should just be an `Option<&str>` if we can't reasonably use it here.